modules/ROOT/pages/enable-inbound-traffic.adoc

```diff
@@ -61,79 +67,107 @@ DNS resolution prior to making the change.
 
 .. Specify the minimum number of cores to allocate for each controller node for the internal load balancer. 
 +
-Note that Runtime Fabric can use up to the maximum number shown if the cores are available. Transport Layer Security (TLS) termination is computationally expensive, so allocate enough CPU to increase throughput and decrease latency. Refer to See the xref:deploy-resource-allocation.adoc[Resource Allocation and Performance on Anypoint Runtime Fabric] for more information on CPU allocation and throughput, including how the TLS private key type and size affect these numbers.
-.. Specify the minimum memory to allocate for each controller node for the internal load balancer. Note that Runtime Fabric can use up to the maximum amount shown if it is available. 
-. Configure a TLS context.
+Note that Runtime Fabric can use up to the maximum number shown if the cores are available. Transport Layer Security (TLS) termination is computationally expensive, so allocate enough CPU to increase throughput and decrease latency. Refer to xref:deploy-resource-allocation.adoc#internal-load-balancer-memory-allocation[Internal Load Balancer Memory Allocation] for more information on CPU allocation and throughput, including how the TLS private key type and size affect these numbers.
+.. Specify the minimum memory to allocate for each controller node for the internal load balancer. Note that Runtime Fabric can use up to the maximum amount shown if it is available.
+
+. In the *Protocol* section, select one of the following protocol options:
+
+* *Accept HTTPS requests…*
++
+Select one of the following options:
+
+** *and redirect HTTP to HTTPS*
 +
-All inbound traffic entering Anypoint Runtime Fabric is encrypted using TLS. When enabling inbound traffic, you must provide a valid TLS certificate. After inbound traffic is enabled, HTTP requests sent to Runtime Fabric receive a `301` response to redirect to HTTPS on port 443.
+After inbound traffic is enabled, HTTP requests sent to Runtime Fabric receive a 301 response to redirect to HTTPS on port 443.
+** *and also accept HTTP requests*
++
+HTTP requests sent to Runtime Fabric are directed to port 80.
```

Comment:
This doesn't sound right. Could we verify w/ engineering?

I think what we mean to say is that, with this option, both HTTPS (port 443) and HTTP (port 80) requests are accepted.